Felienne/hedy
# [BUG FIX] Sharing a program from the programs page now shows correct modal

**Description**
In this PR we change the use of the old deprecated alert modal to the newly used new intrusive one. This was an oversight on the original implementation but should be fix now.

**Fixes**
This PR fixes #2975.

**How to test**
Navigate to the `/programs` page and share any program. Verify that the new alert modal is used instead of the old one.

**Old**
![Schermafbeelding 2022-06-27 om 13 53 10](https://user-images.githubusercontent.com/38155424/175936124-94640f6b-1646-4aa7-a23f-a699d354aef3.png)
**New**
![Schermafbeelding 2022-06-27 om 13 59 59](https://user-images.githubusercontent.com/38155424/175936229-eb60005b-352b-46a4-8f86-a4f063951abc.png)
